### DISCUSSION TITLE
Welcome to parallels-vscode-extension Discussions!

### DISCUSSION THREAD

**Discussion by cjlapao** [Parallels/parallels-vscode-extension#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by toms663** [comment]
trying to load this and when click on the install it says that's its an invalid location.

**Reply by cjlapao** [reply]
Sorrye We will need a bit more information, like screenshots

**Comment by mktpostal** [comment]
The Parallels VS Code extension is great for VM lifecycle management. Starting, stopping, grouping, and handling snapshots are all conveniently accessible within VS Code. However, the “Get Inside the VM” feature is a serious security and usability issue in its current form, because it defaults to logging in as root on Linux or Administrator on Windows.
Ideally, it should prompt for a username and password, then persist the connection for a user‑defined duration. Until that is implemented, this feature poses unnecessary risk and should be reconsidered.
